Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:8-jre
+
+ENV APP_HOME=/fizzbuzz \
+    JAR_FILE=FizzBuzzEnterpriseEdition-all.jar
+RUN mkdir -p ${APP_HOME}
+
+COPY build/libs/${JAR_FILE} ${APP_HOME}
+
+WORKDIR ${APP_HOME}
+
+CMD exec java -jar ${JAR_FILE}
```

Comment:
This should probably also use the new experimental JVM options for cgroup memory limits:

```-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap```

https://bugs.openjdk.java.net/browse/JDK-8170888

Reply:
Done!